packages/openssl.rb

```diff
@@ -3,9 +3,9 @@
 class Openssl < Package
   description 'OpenSSL is an open source project that provides a robust, commercial-grade, and full-featured toolkit for the Transport Layer Security (TLS) and Secure Sockets Layer (SSL) protocols.'
   homepage 'https://www.openssl.org/'
-  version '1.0.2p-2'
-  source_url 'https://github.com/openssl/openssl/archive/OpenSSL_1_0_2p.tar.gz'
-  source_sha256 '95ca65a25bdd41e127e5f4054539e8532a46be602b43b44af7c7100172e7cd50'
+  version '1.0.2q'
+  source_url 'https://github.com/openssl/openssl/archive/OpenSSL_1_0_2q.tar.gz'
```

Comment:
At least use files from `https://openssl.org` instead of using the GitHub mirror.

Reply:
The Github mirror is supported by the project.
```
The master sources are maintained in our git repository, which is accessible over the network and cloned on GitHub, at https://github.com/openssl/openssl.
```